HTTP_200/templates/index.html

```diff
@@ -292,8 +292,8 @@ <h3 class="index-welcome"><span class="index-bars"><i class="fa fa-bars"></i></s
           </div>
         </div>
       </div>
-      <div class="sub-footer ">
-        <h4><a href="http://www.hackncs.com" target="_blank"> Nibble Computer Society</a></h4>
```

Comment:
@aka-jain Can you look into the pull request ? 

@itaditya Thanks for submitting the pull request. Appreciable work. :smile: